data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1680 @@
+	to complete
+		day < 5
+	on complete
+		outfit "Armageddon Core" -1
+		outfit "Fusion Reactor" -1
+		outfit "Breeder Reactor" -1
+		payment 33000000
+		"assisted resistance" ++
+		conversation
+			`When you come in to land, the local Lunarium contacts direct you to a site far away from any of the cities, close to a gigantic cave entrance, on a valley already heavy with the planet's usual rains. Dozens of Arachi and Kimek are waiting for you, using hulking cargo loaders to carefully take each reactor, slowly bringing them inside the cave, and taking several minutes to come back out for the next reactor.`
+			`	Most of those not working on moving the reactors are armed, and keep watch of the skies, occasionally listening into a communicator for any signs of Heliarch ships that may be approaching. After over an hour, once the last reactor is finally taken out of your cargo hold, one of the Arachi comes up to you and hands you <payment>, thanking you for your service to their cause. "A lot of money, this is, but worth it, the reactors should be."`
+			choice
+				`	"Where do you even get all this money from?"`
+				`	"What are you even to do with those reactors? It would be really hard to install them in a ship out here."`
+					goto reactors
+			`	"Word of our existence, the Heliarchs muffle, and distorted and twisted, their versions of our actions are, but enough to drive away our supporters, that is not," he says. "Not entirely... happy, some Arach Houses are, with the direction of things. Help fund us, they do."`
+				goto end
+			label reactors
+			`	"Fit for battle, solar panels are not. Come to fight the Heliarchs, we eventually will, so better power generation for our weapons, we'll need," he says, and then points in the direction of the spaceport. "More efficient, Heliarch reactors are, but too difficult to steal them, it is. Too risky. Only provide energy for our facilities, they do. Studied and reproduced for use in our ships, the reactors you brought us will be."`
+			label end
+			`	They bid you farewell and head into the cave, and you get back inside your ship. Once you land back at the city hangars, the device installed in your monitor beeps again, and you find Chiree messaging you. "Helped us a great deal you have, Captain <last>. Contact you like this, I may not, but find more offers for smuggling jobs, on our planets you may. Appreciated, any more equipment you bring us will be, and of course, pay well we would." She signs off, and the beeps stop.`
+
+
+
+mission "House Bebliss 1-A"
+	name "Eight Legged Hosts"
+	description "Head to <destination> where representatives from an Arach House that supports the Resistance wish to discuss a job offer."
+	minor
+	to offer
+		random < 10
+		has "Coalition: Contributor: done"
+		not "Heliarch License: done"
+		not "Heliarch Investigation 2: active"
+		or
+			has "FW Epilogue: Freya: offered"
```

Comment:
I'm not quite sure why this `or` is here unless you're planning to add something else as option.

Reply:
It's so that when the Navy and Syndicate campaigns are in, their versions of a "Freya remains in Deneb" mission can be checked for here.

Reply:
Perhaps add a comment here to mention that?
I just came here to say what todd said. A comment could save people time in the future.